.github/workflows/_linux-test.yml

```diff
@@ -201,6 +191,16 @@ jobs:
         run: |
           echo "timeout=$((JOB_TIMEOUT-30))" >> "${GITHUB_OUTPUT}"
 
+      - name: Start monitoring script
```

Comment:
I put start monitoring here, but open to discuss : do we also want to log the steps before the test?

Reply:
We don't do anything heavy before the test step, so I think both approach are ok.  But if you move the step closer to the test step here, it makes sense to do the same on other workflows too https://github.com/search?q=repo%3Apytorch%2Fpytorch+%22Start+monitoring+script%22&type=code.

Reply:
Also, this workflow is also used to run all benchmarks.  I remember from our chat last week that you want to leave benchmark for now because it might affect the results.  Is this still the case?

Reply:
sounds good, i will move all of the monitoring script in other test ymls in next pr